client/src/App.tsx

```diff
@@ -30,17 +30,35 @@ import { UserCursor } from './components/UserCursors';
 import _ from 'underscore';
 // Sound imports
 //@ts-ignore
-import audioEnter from './assets/sounds/zap-enter.mp3';
-//@ts-ignore
-import audioExit from './assets/sounds/zap-exit.mp3';
-//@ts-ignore
-import cymbalHit from './assets/sounds/cymbal.mp3';
-//@ts-ignore
-import drumBeat from './assets/sounds/drumbeat.mp3';
-//@ts-ignore
-import gotEm from './assets/sounds/ha-got-eeem.mp3';
-//@ts-ignore
-import guitarStrum from './assets/sounds/electric_guitar.mp3';
+import audioEnter from './assets/sounds/zap-enter.mp3'; //@ts-ignore
+import audioExit from './assets/sounds/zap-exit.mp3'; //@ts-ignore
+
+// Instruments
+import cymbalHit from './assets/sounds/instruments/cymbal.mp3'; //@ts-ignore
+import drumBeat from './assets/sounds/instruments/drumbeat.mp3'; //@ts-ignore
+import trumpet from './assets/sounds/instruments/trumpet.mp3'; //@ts-ignore
+import guitarStrum from './assets/sounds/instruments/electric_guitar.mp3'; //@ts-ignore
+import gong from './assets/sounds/instruments/chinese-gong.wav'; //@ts-ignore
+import harp from './assets/sounds/instruments/harp.wav'; //@ts-ignore
+// Funny
+import gotEm from './assets/sounds/funny/ha-got-eeem.mp3'; //@ts-ignore
+import noice from './assets/sounds/funny/noice.mp3'; //@ts-ignore
+import stop_it_get_some_help from './assets/sounds/funny/stop_it_get_some_help.mp3'; //@ts-ignore
+import ahh from './assets/sounds/funny/ahh.mp3'; //@ts-ignore
+import air from './assets/sounds/funny/air.mp3'; //@ts-ignore
+import applause from './assets/sounds/funny/applause.mp3'; //@ts-ignore
+import clang from './assets/sounds/funny/clang.mp3'; //@ts-ignore
+import groan from './assets/sounds/funny/groan.mp3'; //@ts-ignore
+import horn from './assets/sounds/funny/horn.mp3'; //@ts-ignore
+import laugh from './assets/sounds/funny/laugh.mp3'; //@ts-ignore
+// Nature
+import bee from './assets/sounds/nature/bee.mp3'; //@ts-ignore
+import dog from './assets/sounds/nature/dog.mp3'; //@ts-ignore
+import flying_fox from './assets/sounds/nature/flying-fox.mp3'; //@ts-ignore
+import lightning from './assets/sounds/nature/lightning.mp3'; //@ts-ignore
+import nature from './assets/sounds/nature/nature.mp3'; //@ts-ignore
+import sealion from './assets/sounds/nature/sealion.mp3'; //@ts-ignore
+import water from './assets/sounds/nature/water.mp3';
```

Comment:
this is super long, should import this in another file and just import the sounds: ISound object